third_party/lua-cjson/lua_cjson.c

```diff
@@ -52,6 +52,8 @@
 #include "mp_extension_types.h" /* MP_DECIMAL, MP_UUID */
 #include "diag.h"
 #include "tt_static.h"
+#include "core/tt_uuid.h" /* tt_uuid_to_string(), UUID_STR_LEN */
```

Comment:
How is tt_uuid.h related to this patchset?

Reply:
Good catch! Deleted. Thanks!